src/main/resources/locale_de_DE.properties

```
@@ -105,6 +105,10 @@ command.reputation.profile.title=Reputation von %NAME%
 command.roles.description=Verwaltet die Reputationsrollen.
 command.roles.sub.add=Fügt eine Reputationsrolle hinzu.
 command.roles.sub.add.added=Die Rolle %ROLE% mit der benötigten Reputation %POINTS% wurde erfolgreich hinzugefügt.
+command.roles.sub.addDonor=Füge eine Spenderrolle hinzu, welche Reputation erhalten kann
+command.roles.sub.addDonor.add=Spender Rolle %ROLE% hinzugefügt
+command.roles.sub.addReceiver=Füge eine Empfängerrolle hinzu, welche Reputation vergeben kann
+command.roles.sub.addReceiver.add=Empfänger Rolle %ROLE% hinzugefügt
```

`Empfänger-Rolle` sounds a little bit odd but I'm not sure if there is a better alternative. 
```suggestion
command.roles.sub.addDonor=Fügt eine Spender-Rolle hinzu, welche Reputation erhalten kann.
command.roles.sub.addDonor.add=Spender-Rolle %ROLE% hinzugefügt.
command.roles.sub.addReceiver=Füge eine Empfänger-Rolle hinzu, welche Reputation vergeben kann.
command.roles.sub.addReceiver.add=Empfänger-Rolle %ROLE% hinzugefügt.
```